index.html

```diff
@@ -35,9 +35,157 @@ <h1 class="headline_top">I'm Sergiy<br>Glad to see you!</h1>
         </li>
       </ul>
     </section>
-    <!-- placeholder: start other pages -->
+    <!-- work 1 -->
     <section> 
-      <p></p>
+      <div class="work-container">
+        <div class="work-item">
+          <section class="work-item-flex">
+            <img src="./img/work-snapshot1.png" alt="work1">
+            <header class="work-item-title">Tonic</header>
+            <ul class="work-item-frame">
+              <li class="work-item-frame-client">CANOPY</li>
+              <li class="work-item-frame-counter">●</li>
+              <li class="work-item-frame-role">Back End Dev</li>
+              <li class="work-item-frame-counter">●</li>
+              <li class="work-item-frame-year">2015</li>
+            </ul>
+            <p class="work-item-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="work-item-tags">
+              <li class="work-item-tag">html</li>
+              <li class="work-item-tag">css</li>
+              <li class="work-item-tag">javaScript</li>
+            </ul>
+            <button class="work-item-button">See Project</button>
```

Comment:
- [ ] As the previous reviewer said, kindly make sure you follow the Figma design sizing, and fonts properly for a better result. As I can see, you did nit give the work-item-text class font-size at all.
## Your design
![sc](https://user-images.githubusercontent.com/86200356/191918084-118a86f8-f73d-4947-b4f9-6dbfe453d429.png)

## Figma Design
![image](https://user-images.githubusercontent.com/86200356/191918369-25493989-1f4b-4db6-b849-cd86405293e4.png)

Reply:
I've set font-size as 250% to keep relative design. From my end I see the proper site's view on 375px width and even smaller. Hope it's OK now.